ERCS/erc-x.md

```diff
@@ -0,0 +1,324 @@
+---
+eip: 7929
+title: PermaLink Asset Bound Token Standard
+description: An interface for PermaLink Asset Bound Tokens, also known as a PermaLink-ABTs.
```

Comment:
Your description restates a bit too much of the title. You want to use this field to expand on the ideas introduced in your title instead of just repeating the same information.

```suggestion
description: Asset-bound Tokens (ABT) permanently bound to another token, known as PermaLink-ABTs.
```

How's that?